sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FieldValue.java

```diff
@@ -196,4 +201,19 @@ public String asPhoneNumber() {
         }
         return this.formFieldMap;
     }
+
+    /**
+     * Gets the value of the field as a selection mark.
+     *
+     * @return the value of the field as an unmodifiable selection mark.
+     * @throws UnsupportedOperationException if {@link FieldValue#getValueType()} is not
+     * {@link FieldValueType#SELECTION_MARK}.
+     */
+    public FormSelectionMark asSelectionMark() {
```

Comment:
According to how the type is populated, this might be misleading. Here we only have the state of the selection mark, but we are returning a whole `FormSelectionMark`.
How are we getting the rest of the information for that object?

Reply:
yes, this should return the SelectionMarkState.